package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "blaze-elements",
+  "name": "@blaze-elements/blaze-elements",
```

Comment:
Is it really wise to have main package un-scoped & all sub-packages scoped?
- It does not make a lot of sense...